docs/introduction.rst

```diff
@@ -0,0 +1,111 @@
+Introduction
+============
+
+The XAITK-Saliency toolkit implements a class of XAI algorithms known
+as `saliency algorithms`. The basic application pipeline is shown in Figure 1:
+
+.. figure:: figures/intro-fig-01.png
+
+*Figure 1: A basic AI pipeline.*
+
+In this scenario, an AI algorithm operates on an input (text, image,
+etc.) to produce some sort of output. Saliency algorithms build on
+this as shown in Figure 2:
+
+.. figure:: figures/intro-fig-02.png
+
+*Figure 2: The AI pipeline augmented with a saliency algorithm.*
+
+At a high level, saliency maps are typically a copy of the input with
+regions highlighted to indicate areas that are somehow significant to
+the AI. Figure 3 shows sample saliency maps for text and images.
+
+.. figure:: figures/intro-fig-03.png
+
+*Figure 3: Sample saliency maps for (left) text (from* `Tuckey et al.
+<https://arxiv.org/abs/1907.05664>`_) *and (right) images (from* `Dong et
+al. <https://openaccess.thecvf.com/content_CVPRW_2019/html/Explainable_AI/Dong_Explainability_for_Content-Based_Image_Retrieval_CVPRW_2019_paper.html>`_
+*)*
+
+.. note:: The XAITK-Saliency toolkit focuses on providing saliency
+          maps for images.
+
+Image Saliency Maps: An Intuitive Introduction
+----------------------------------------
+
+Figure 4 shows a deep learning pipeline for recognizing objects in
+images; pixels in the image (in green) are processed by the pipeline
+to produce the output (in orange). Here, the system has been trained
+to recognize 1000 object types. Its output is a list of 1000 numbers,
```

Comment:
`types` -> `categories`, just to be slightly more consistent with the literature.

Reply:
Yes. Good catch.